packages/contentful--react-apps-toolkit/src/useCMA.ts

```diff
@@ -1,12 +1,13 @@
-import { createClient, PlainClientAPI } from 'contentful-management';
+import { createClient } from 'contentful-management';
+import { CMAClient } from '@contentful/app-sdk';
 import { useMemo } from 'react';
 import { useSDK } from './useSDK';
 
 /**
  * React hook returning a CMA plain client instance.
```

Comment:
Should we mention that the type differs from `PlainClientAPI`? Not quite sure - also fine with leaving it as it is.